core/internal/features/features_test.go

```diff
@@ -631,7 +646,7 @@ observationSource   = """
 		require.Len(t, outputs, 1)
 		output := outputs[0]
 		receipt := output.(map[string]interface{})
-		assert.Equal(t, "0x11", receipt["blockNumber"])
+		assert.Equal(t, "0x13", receipt["blockNumber"])
```

Comment:
Did we change testing behaviour?

Reply:
I did to account for the gap between confirmation and receipt fetch. Since we can't resume task runs in the same block that we mark a transaction as confirmed, we have to commit new heads to trigger the finalizer loop to find/store receipts and resume tasks.